Assets/SEE/Game/Portal.cs

```diff
@@ -78,27 +73,25 @@ public static void GetPortal(GameObject gameObject, out Vector2 leftFront, out V
         }
 
         /// <summary>
-        /// Returns the portal of <paramref name="material"/> a rectangle in the x/z plane
-        /// with <paramref name="leftFront"/> corner and <paramref name="rightBack"/> corner.
-        ///
-        /// Precondition: <paramref name="material"/> must have a the attributes
-        /// <see cref="portalMin"/> and <see cref="portalMax"/>. If they do not
-        /// exist, the result is undefined.
+        /// Returns the portal of <paramref name="material"/>, a rectangle on the x/z plane,
+        /// defined by its <paramref name="leftFront"/> and <paramref name="rightBack"/> corners.
+        /// <para>
+        /// Precondition: <paramref name="material"/> must have the <c>_Portal</c> attribute
+        /// If it does not exist, the result is undefined.
+        /// </para>
         /// </summary>
         /// <param name="gameObject">game objects whose portal is requested</param>
```

Comment:
This isn't part of your changes, but the parameter documentation does not match the actual parameter.